### ISSUE TITLE
Content of HamburgerMenu.HeaderContent not visible

### ISSUE THREAD

**Issue by jp-weber** [Windows-XAML/Template10#875]
The content of the HamburgerMenu.HeaderContent is not visible because the Background of HamburgerButtonGrid blends it. My workaround is to set the Background of HamburgerButtonGrid to transparent and give the HamburgerButton a color.

```
<Grid x:Name="HamburgerButtonGrid"
              Grid.Row="0"
              Width="{x:Bind HamburgerButtonGridWidth, Mode=OneWay}"
              HorizontalAlignment="Left"
              VerticalAlignment="Top"
              Background="Transparent"
              Canvas.ZIndex="123"
              Visibility="{x:Bind HamburgerButtonVisibility, Mode=OneWay}">
            <Button x:Name="HamburgerButton"
                    AutomationProperties.Name="Hamburger Menu Button"
                    Background="{x:Bind HamburgerBackground, FallbackValue=DarkRed, Mode=OneWay}"
                    Command="{x:Bind HamburgerCommand}"
                    Foreground="{x:Bind HamburgerForeground, FallbackValue=White, Mode=OneWay}"
                    RenderTransformOrigin="0.5,0.5"
                    Style="{StaticResource HamburgerButtonStyle}">
                <FontIcon FontSize="20" Glyph="&#xE700;" />
            </Button>
        </Grid>
```

In the Hamburger Sample/Shell.xaml the code is:

```
<Controls:HamburgerMenu.HeaderContent>
            <Grid Height="48" Background="{ThemeResource CustomColorBrush}">
                <TextBlock Text="Minimal Header" FontSize="16" VerticalAlignment="Center" Margin="12,0,0,0"/>
            </Grid>
        </Controls:HamburgerMenu.HeaderContent>
```

**Comment by JerryNixon** [comment]
Submit a pull request so I can see what you mean.

**Comment by jp-weber** [comment]
fix with #891
